sources/Interop/Windows/other/helper-types/COLORREF.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+using System;
+
+namespace TerraFX.Interop
+{
+    public partial struct COLORREF : IEquatable<COLORREF>
+    {
+        public readonly uint Value;
+
+        public COLORREF(uint value)
+        {
+            Value = value;
+        }
+
+        public static bool operator ==(COLORREF left, COLORREF right) => left.Value == right.Value;
+
+        public static bool operator !=(COLORREF left, COLORREF right) => left.Value != right.Value;
+
+        public static implicit operator COLORREF(byte value) => new COLORREF((uint)(value));
+
+        public static explicit operator byte(COLORREF value) => (byte)(value.Value);
+
+        public static implicit operator COLORREF(short value) => new COLORREF((uint)(value));
+
+        public static explicit operator short(COLORREF value) => (short)(value.Value);
+
+        public static explicit operator COLORREF(int value) => new COLORREF((uint)(value));
+
+        public static explicit operator int(COLORREF value) => (int)(value.Value);
+
+        public static explicit operator COLORREF(long value) => new COLORREF((uint)(value));
+
+        public static implicit operator long(COLORREF value) => (long)(value.Value);
+
+        public static explicit operator COLORREF(nint value) => new COLORREF((uint)(value));
+
+        public static implicit operator nint(COLORREF value) => (nint)(value.Value);
+
+        public static implicit operator COLORREF(sbyte value) => new COLORREF((uint)(value));
+
+        public static explicit operator sbyte(COLORREF value) => (sbyte)(value.Value);
+
+        public static implicit operator COLORREF(ushort value) => new COLORREF((uint)(value));
+
+        public static explicit operator ushort(COLORREF value) => (ushort)(value.Value);
+
+        public static implicit operator COLORREF(uint value) => new COLORREF((uint)(value));
+
+        public static implicit operator uint(COLORREF value) => (uint)(value.Value);
+
+        public static explicit operator COLORREF(ulong value) => new COLORREF((uint)(value));
+
+        public static implicit operator ulong(COLORREF value) => (ulong)(value.Value);
+
+        public static explicit operator COLORREF(nuint value) => new COLORREF((uint)(value));
+
+        public static implicit operator nuint(COLORREF value) => (nuint)(value.Value);
+
+        public override bool Equals(object? obj) => (obj is COLORREF other) && Equals(other);
+
+        public bool Equals(COLORREF other) => (this == other);
+
+        public override int GetHashCode() => Value.GetHashCode();
+
+        public override string ToString() => Value.ToString();
```

Comment:
this one could definitely benefit from hex formatting, since colors are often conceptually `#RRGGBB` or `#AARRGGBB`